### PR TITLE
Disable systemd service file creation

### PR DESCRIPTION
**Pull request by witekest**
'skip_enable' option controlls the creation of systemd service file for
Monasca Agent. The service file is created in [RPM package](https://review.openstack.org/436063), so the option has to be enabled here.

### CODE REVIEW

File: chef/data_bags/crowbar/template-monasca.json [hunk 1]
```diff
@@ -24,7 +24,8 @@
         "max_buffer_size": 1000,
         "max_measurement_buffer_size": -1,
         "backlog_send_rate": 1000,
-        "amplifier": 0
+        "amplifier": 0,
+        "skip_enable": true
```
Comment (kornicameister) [inline chef/data_bags/crowbar/template-monasca.json]: I wonder if, assuming that service file is created in RPM package, this variable should be added here at all.
Why just not enforce it ?

**Review by kornicameister — CHANGES_REQUESTED** [review]

**Comment by matrixik** [comment]
Wont it be actually better to overwrite this file every time (every 15 min) `monasca-reconfigure` is run? Then if there are newer version of monasca-agent updated in rpms then it will provide all fixes to service files without need to update them in rpm-packaging (beside updating agent version in spec file).

But if you think it's better to add it only once then I'm fine with it but I think it should be enforced to not overwrite it and not provide option to configure it.

**Comment by witekest** [comment]
Service configuration files are normally controlled by the RPM package, so I would prefer to keep it there. I will enforce 'skip_enable' option now.

File: chef/data_bags/crowbar/template-monasca.json (hunk 1 above)
Comment (witekest) [reply chef/data_bags/crowbar/template-monasca.json]: Done.

**Review by kornicameister — APPROVED** [review]

**Review by matrixik — APPROVED** [review]

**Pull request merged**